build-prod.sh

```diff
@@ -4,6 +4,9 @@ set -e
 
 IMAGE_NAME="quay.io/ipfs/jenkins"
 
+# Making sure jenkins-secrets are not included in the image
+git submodule deinit .
```

Comment:
Add `-f` here -- "If --force is specified, the submodule’s work tree will be removed even if it contains local modifications." We care mostly about the work tree, and not so much about the submodule config.